cms/djangoapps/contentstore/tests/test_clone_course.py

```diff
@@ -46,6 +54,57 @@ def test_clone_course(self):
             self.store.clone_course(split_course3_id, split_course4_id, self.user.id)
             self.assertCoursesEqual(split_course3_id, split_course4_id)
 
+    def test_space_in_asset_name_for_rerun_course(self):
+        """
+        Tests check the scenerio where one course which has an asset with percentage(%) in its
+         name, it should re-run successfully.
+        """
+        org = 'edX'
+        course_number = 'CS101'
+        course_run = '2015_Q1'
+        fields = {'display_name': 'rerun'}
+        course_assets = set([u'subs_Introduction%20To%20New.srt.sjson', u'subs_vhas5o_fW-8.srt.sjson'])
+
+        # Import a course using split modulestore
+        with modulestore().default_store(ModuleStoreEnum.Type.split):
+            module_store = modulestore()
+            content_store = contentstore()
+            course_id = module_store.make_course_key(org, course_number, course_run)
+
+            import_course_from_xml(
```

Comment:
instead of importing a new course, can you create a course with `CourseFactory` and assets to it from there?